lib/FieldType/DataTransformer/CheckboxValueTransformer.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryForms\FieldType\DataTransformer;
+
+use eZ\Publish\Core\FieldType\Checkbox\Value;
+use Symfony\Component\Form\DataTransformerInterface;
+
+class CheckboxValueTransformer implements DataTransformerInterface
```

Comment:
I'm thinking that those transformers (Checkbox, ISBN, Integer, Float) could be replaced with a unique `SimpleValueTransformer` (not the best name). It would look like this:

```php
<?php
use Symfony\Component\Form\DataTransformerInterface;

class SimpleValueTransformer implements DataTransformerInterface
{
    private $valueClass;
    
    public function __construct($valueClass)
    {
        $this->valueClass = $valueClass;
    }

    public function transform($value)
    {
        if (!$value instanceof $this->valueClass) {
            return null;
        }

        return $value->value;
    }
    public function reverseTransform($value)
    {
        if (!$value) {
            return null;
        }
        
        return new $this->valueClass($value);
    }
}
```

A mapper would use it like this:
```php
->addModelTransformer(new SimpleValueTransformer(CheckboxValue::class))
```

It would allow us to replace four classes with one that covers many simple cases.

What do you think ?